calico_containers/tests/st/docker_host.py

```diff
@@ -85,3 +84,58 @@ def assert_driver_up(self):
         sock_exists = partial(self.execute,
                               "[ -e %s ]" % CALICO_DRIVER_SOCK)
         retry_until_success(sock_exists, ex_class=CalledProcessError)
+
+    def remove_containers(self):
+        """
+        Remove all containers running on this host.
+
+        Useful for test shut down to ensure the host is cleaned up.
+        :return: None
+        """
+        # TODO: only remove ST created containers for non-dind.
+        cmd = "docker rm -f $(docker ps -qa) ; docker rmi $(docker images -qa)"
+        ok_codes = [0,
```

Comment:
Erm, probably.  But, this code is pythonized in master, so I'd like to sort it out after merging to master.